src/style.css

```diff
@@ -0,0 +1,87 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+form {
+  position: relative;
+  margin-top: 20px;
+}
+
+[data-qa="big"] input {
+  box-sizing: border-box;
+  padding-left: 62px;
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  font-family: Avenir;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+[data-qa="small"] input {
+  box-sizing: border-box;
+  padding-left: 33px;
+  width: 100%;
+  height: 42px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  font-family: Avenir;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
```

Comment:
You have a lot of repeatable styles. 
So you need to refactor you code. I will give you an EXAMPLE, how to do it

Following the BEM-methodology - use the `block` selector for the FIRST input (`class="input"`), where you apply common input styles

```
.input {
  {{common_input_styles}}
}
```

And then you apply `modifier` to the SECOND input (`class="input input--small"`) that have ONLY differences with base class input
```
.input--small {
  {{styles_with_difference}}
}
```
So in this way you follow DRY (Don't repeat yourself) principe
Apply this principe all over the styles